### DISCUSSION TITLE
[Enhancement] Make note in repository settings about Emoji skin tone being visible publicly

### DISCUSSION THREAD

**Discussion by seanpm2001** [community/community#106369] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

***

I feel there needs to be a small enhancement here. The preferred emoji skin tone selection menu does not specify that the preference is public and appears on your profile (notably through GitHub achievements) I feel a line like this should fix the problem:

<details open><summary>Image</summary>

![Screenshot 2024-02-03 at 19-54-24 Build software better together](https://github.com/community/community/assets/65933340/a56831a4-066a-4602-b0d1-5bd1c65617b1)

:octocat: [`Alt image (if the previous image does not load)`](https://github.com/seanpm2001/seanpm2001/blob/master/External_Commits/!Discussions/community/!Media/2024.02.05/GitHub_Emoji_Skin_Tone_Preference.png)

</details>

```html
<p><b>Note:</b> Changes here will be made visible on your public GitHub profile.</p>
```

The text and source code should be reworked, as a disclaimer is necessary here.

***

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation

Acces the [Github Support](https://support.github.com/)

Click on My Ticket :

![github04](https://github.com/community/community/assets/72712095/096b3477-cee6-4a19-9f1b-6fb2219b51a1)

and open a New Ticket:

![github05](https://github.com/community/community/assets/72712095/7413fc55-875a-498c-93eb-5c54d5ffc8d6)


God bless you!

**Reply by seanpm2001** [reply]
I just went and posted the ticket. I will leave a new response here if anything comes of it.

**Comment by seanpm2001** [comment]
Latest update (GitHub support response)

> Hi Sean,
> 
> Thanks for getting in touch with GitHub Support.
> 
> Improvements and new features for GitHub profiles and their settings are currently deprioritised in respect to other initiatives. 
> 
> At this time the community discussions forum post you've already made is the best place for your change request, so other community members can share their thoughts too and help your request garner priority.
> 
> I might suggest including some context that explains your concern with this setting being apparent on user profiles - its purpose is to affect the public-facing usage of emojis in comments and reactions. 
> If you need anything else, just let me know.
> Cheers
> Andy
